projects/plugins/jetpack/extensions/blocks/subscriber-login/subscriber-login.php

```diff
@@ -110,15 +110,18 @@ function is_subscriber_logged_in() {
 function render_block( $attributes ) {
 	Jetpack_Gutenberg::load_assets_as_required( __DIR__ );
 
-	$block_template      = '<div %1$s><a href="%2$s">%3$s</a></div>';
-	$redirect_to_current = isset( $attributes['redirectToCurrent'] ) && $attributes['redirectToCurrent'];
+	$block_template             = '<div %1$s><a href="%2$s">%3$s</a></div>';
+	$redirect_to_current        = ! empty( $attributes['redirectToCurrent'] );
+	$log_in_label               = ! empty( $attributes['logInLabel'] ) ? $attributes['logInLabel'] : __( 'Log in', 'jetpack' );
+	$log_out_label              = ! empty( $attributes['logOutLabel'] ) ? $attributes['logOutLabel'] : __( 'Log out', 'jetpack' );
+	$manage_subscriptions_label = ! empty( $attributes['manageSubscriptionsLabel'] ) ? $attributes['manageSubscriptionsLabel'] : __( 'Manage subscriptions', 'jetpack' );
```

Comment:
Something like this might make sense here:

https://github.com/Automattic/jetpack/blob/bd4b3ed9532d80ab495fbe0f59e8c0c2ac141d84/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L973-L985

Reply:
Let's also use `esc_html__()` in case.

Reply:
Nice catch, thanks! Decided to use `sanitize_text_field` on custom labels which seems easier to use.

Reply:
> Let's also use esc_html__() for translation functions in case.

Not sure if it's necessary since we put only static texts into the translation function without any extra params. Still good to know such a function exists, thanks!

Reply:
> Not sure if it's necessary since we put only static texts into the translation function without any extra params. Still good to know such a function exists, thanks!

If you look at the code, we do use those quite a lot when we don't expect HTML in strings. I think it's a security measure against malicious translators. I.e. no reason not to.

Reply:
Sure, added 😄